cachito/workers/scm.py

```diff
@@ -143,7 +143,7 @@ def _create_archive(self, from_dir):
             os.unlink(self.sources_dir.archive_path)
             raise
 
-    def clone_and_archive(self, gitsubmodule=False):
+    def clone_and_archive(self, gitsubmodule=False, include_git_dir=False):
```

Comment:
Can we add a line about `include_git_dir` param?

Reply:
You mean in the docstring? Sure, I missed that one.